.github/workflows/cluster-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: Cluster tests ( openjdk11 & mysql )
+name: Cluster tests (Ubuntu, OpenJDK11 & MySQL)
```

Comment:
Ubuntu will already be there "Cluster tests (Ubuntu, OpenJDK11 & MySQL) / ubuntu (pull_request) " 
It is redundant.